#arduino_input.msg
int8 mode
float32 data1
float32 data2


